network/codec/json/decode.go

```diff
@@ -20,74 +22,74 @@ func switchenv2v(env Envelope) (interface{}, error) {
 	switch env.Code {
 
 	// consensus
-	case CodeBlockProposal:
+	case codec.CodeBlockProposal:
```

Comment:
This is great. I'm happy you're aligning on using a single set of consts for all codecs. 

The original json and cbor consts were offset by 1. Do you know if it's safe to change the value? i.e. is the data stored anywhere, or is it only used in transit so an update at a spork boundary would be safe?

Reply:
A reasonable concern to consider in the next TPM.

Reply:
https://github.com/onflow/flow-go/pull/2491#discussion_r886048382